addons/sourcemod/scripting/include/hosties.inc

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
do we really need too have a dupe of this file , do u know where it exist on some another repo?
Same for lastrequest.inc